### DISCUSSION TITLE
Why is disabling suspense recommended?

### DISCUSSION THREAD

**Discussion by StevieJayCee** [sergiodxa/remix-i18next#121] [Q&A] [answered]
I see as part of the documentation; is there any more context around the reasoning of this option? Is the recommendation because of remix,  i18Next or the state of suspense at that time?

```
  // Disabling suspense is recommended
  react: { useSuspense: false },
```

**Comment by sergiodxa** [comment, marked as answer]
Because when I wrote that React 18 was not released yet and in React 17 you can't use suspense with SSR.

If you're on React 18 feel free to enable Suspense.

**Reply by StevieJayCee** [reply]
Thanks @sergiodxa

**Reply by ben-wfp** [reply]
As mentioned by someone in a comment on [this thread](https://github.com/sergiodxa/remix-i18next/discussions/66), hydration wouldn't work for me _unless_ I removed `{ useSuspense: false }`
